### ISSUE TITLE
backup-fetch problem: failed to fetch sentinel: context canceled

### ISSUE THREAD

**Issue by zloichel** [wal-g/wal-g#400]
Hello!
I have problem with backup-fetch from google storage:
```
sudo -u postgres /opt/wal-g/run.sh backup-fetch test base_000000030000013F000000BE
DEBUG: 2019/09/04 12:07:59.680996 HandleBackupFetch(base_000000030000013F000000BE, folder, test)
ERROR: 2019/09/04 12:08:00.543118 Failed to fetch backup: failed to fetch sentinel: context  canceled
```
backup-list and backup-push worked fine, service-account has full access to google storage.
i tried all backups in backup-list - some error...
```

sudo -u postgres /opt/wal-g/run.sh backup-list
name                          last_modified        wal_segment_backup_start
base_000000030000013D000000EA 2019-08-14T09:29:59Z 000000030000013D000000EA
base_000000030000013F000000BE 2019-08-15T02:21:42Z 000000030000013F000000BE
base_000000030000014300000045 2019-08-16T02:22:54Z 000000030000014300000045
base_0000000300000145000000BC 2019-08-17T02:22:32Z 0000000300000145000000BC
base_00000003000001480000001A 2019-08-18T02:22:04Z 00000003000001480000001A
base_000000030000014A0000006B 2019-08-19T02:22:02Z 000000030000014A0000006B
base_000000030000014C000000EE 2019-08-20T02:23:34Z 000000030000014C000000EE
base_000000030000014F0000003A 2019-08-21T02:23:53Z 000000030000014F0000003A
base_0000000300000151000000C8 2019-08-22T02:23:27Z 0000000300000151000000C8
base_00000003000001540000002F 2019-08-23T02:24:26Z 00000003000001540000002F
base_00000003000001560000008C 2019-08-24T02:24:17Z 00000003000001560000008C
base_0000000300000157000000F9 2019-08-25T02:24:28Z 0000000300000157000000F9
base_000000030000015900000030 2019-08-26T02:24:32Z 000000030000015900000030
base_000000030000015A0000007B 2019-08-27T02:24:21Z 000000030000015A0000007B
base_000000030000015C00000006 2019-08-28T02:24:42Z 000000030000015C00000006
base_000000030000015D00000071 2019-08-29T02:25:07Z 000000030000015D00000071
base_000000030000015E000000EB 2019-08-30T02:25:00Z 000000030000015E000000EB
base_0000000300000160000000AC 2019-08-31T02:24:19Z 0000000300000160000000AC
base_000000030000016200000033 2019-09-01T02:24:00Z 000000030000016200000033
base_0000000300000163000000A6 2019-09-02T02:23:52Z 0000000300000163000000A6
base_000000030000016500000034 2019-09-03T02:23:29Z 000000030000016500000034
base_0000000300000166000000AB 2019-09-04T02:23:16Z 0000000300000166000000AB

```

in run.sh only environment variables:

```
#!/bin/bash

PATH=/usr/bin:/usr/local/bin:$PATH

if [[ -f /opt/wal-g/environment ]]; then
        source /opt/wal-g/environment
fi

/usr/local/bin/wal-g $@
```

wal-g release = 0.2.11

**Comment by zloichel** [comment]
when i copied base_000000030000016500000034 on disk  - backup-fetch runned succesfully (with WALG_FILE_PREFIX to this folder).

**Comment by x4m** [comment]
Hi! Thanks for the report!
The issue is fixed in #399 . If you need workaraound ASAP, plz use release 0.2.9.
The bug was introduced in https://github.com/wal-g/wal-g/commit/c18522e948bdb7dfb00f3688d48273d8ddcde7cf

**Comment by zloichel** [comment]
its worked, tnx!
